src/wrapped_radix_engine_toolkit/low_level/transaction_manifest/blobs/blobs.rs

```diff
@@ -6,6 +6,29 @@ pub struct Blobs {
     pub(crate) secret_magic: BlobsSecretMagic,
 }
 
+impl Serialize for Blobs {
```

Comment:
Try remove this manual impl of `Serialize` and `Deserialize` and use derive with `#[serde(transparent)]` on all nodes in the chain (on all secret magic types)

Reply:
If you need special behaviour in case of `"null"` try out other serde macro attributes:

https://stackoverflow.com/a/77733371/1311272

```
#[serde_as]
#[derive(Serialize, Deserialize)]
pub struct Element {
    #[serde_as(as = "DefaultOnNull")]
    color: Option<String>
}
```